test/TestObjects.sol

```diff
@@ -1,28 +1,32 @@
-pragma solidity ^0.4.5;
+pragma solidity ^0.4.23;
 
 import "truffle/Assert.sol";
 import "../contracts/JsmnSolLib.sol";
 
 contract TestObjects {
-
+    
     uint constant RETURN_SUCCESS = 0;
     uint constant RETURN_ERROR_INVALID_JSON = 1;
     uint constant RETURN_ERROR_PART = 2;
     uint constant RETURN_ERROR_NO_MEM = 3;
-
+    
     function testSimpleObject() public {
-        string memory json = '{"outerKey": {"innerKey": "value"}}';
-
+        string memory json = '{"supplier_company_name":"Item description","supplier_authorized_person_name":"100","supplier_authorized_person_email":"Ton","supplier_company_street":"Steel Work","supplier_company_mobile_phone":"EUR900","supplier_company_fax":"EUR4500","supplier_account_beneficiary_name":"shuja","supplier_bank_name":"Emirates NBD"}';
```

Comment:
What's the reason that you've extended the simple test to such a long JSON?

Reply:
agreed. just made it short as you recommended. I added it based on the project I was using the library in.